client/src/views/files/handler/FileHandler.vue

```diff
@@ -239,15 +239,18 @@ const cancelRouteWatch = watchRoute(async () => {
   }
 
   const query = getCurrentRouteQuery();
+
   const timestamp = Number.isNaN(Number(query.timestamp)) ? undefined : Number(query.timestamp);
+  const fileHandlerMode = getFileHandlerMode();
 
   // Same file, no need to reload
   if (contentInfo.value && contentInfo.value.path === getDocumentPath() && atDateTime.value?.toMillis() === timestamp) {
-    const fileHandlerMode = getFileHandlerMode();
     if (fileHandlerMode === handlerMode.value) {
       return;
     }
     handlerMode.value = getFileHandlerMode();
+  } else if (fileHandlerMode !== handlerMode.value) {
+    handlerMode.value = getFileHandlerMode();
```

Comment:
Why not use `fileHandlerMode` const instead of calling `getFileHandlerMode()` again? (same for line 253)

Reply:
Yep, you're right. I refactored it a bit in the process.